crates/proto/src/rr/record_data.rs

```diff
@@ -167,6 +167,10 @@ pub enum RData {
     /// ```
     CNAME(Name),
 
+    /// ```text
```

Comment:
Can you add the text that talks about it's use in DNS?

Reply:
Absolutely, copied in some text from the RFC!